code/modules/mob/logout.dm

```diff
@@ -15,4 +14,6 @@
 			var/datum/callback/CB = foo
 			CB.Invoke()
 
+	qdel(GetComponent(/datum/component/moved_relay))
```

Comment:
Don't we already have a wrapper for removing a components?
And anyway, why don't we keep the component and reuse it again when the client reconnects?

Reply:
You could do
```dm
var/datum/component/moved_relay/component = GetComponent(/datum/component/moved_relay)
component.Remove()
```
but that doesn't qdel it.
As for reusing the component, it is not worth the effort for the minimal gain, it would need extra logic to handle moving things which don't have clients, which adds overhead anyway and the cost of deleting and adding is pretty minimal since its triggered in rare situations. This just isn't worth me adding and would be a waste of time.